package dev.rage4j.asserts.exception;

public class Rage4JFaithfulnessException extends RuntimeException
{
	public Rage4JFaithfulnessException(String message)
	{
		super(message);
	}

	public Rage4JFaithfulnessException()
	{
	}
}
